src/accessibility.md.njk

```diff
@@ -38,12 +38,9 @@ The Design System and Frontend were introduced in June 2018 to replace the follo
 
 The GOV.UK Design System team no longer supports these products and will not be making updates to help them meet level AA of WCAG 2.1. 
 
-If you are using these products, it’s recommended that you begin [updating your service now to use the Design System and Frontend](https://design-system.service.gov.uk/get-started/updating-your-code/).
+If you are using these products, you should start [updating your service to use the Design System and Frontend](https://design-system.service.gov.uk/get-started/updating-your-code/). Read guidance on [how to get started](https://design-system.service.gov.uk/get-started/).
 
-Doing this will allow you to update your service in a much more manageable way, than waiting to update straight to Frontend v.3.0.0.
+If you have any questions or need help, you can contact the GOV.UK Design System team:
 
-If you would like to start using the Design System and Frontend, read guidance on [how to get started](https://design-system.service.gov.uk/get-started/).
-
-The GOV.UK Design System team will keep this page up to date with updates and when the new version of Frontend is available.
-
-If you have any questions in the meantime, [get in touch](https://design-system.service.gov.uk/get-in-touch/).
+- on [#govuk-design-system channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system)
+- by email at govuk-design-system-support@digital.cabinet-office.gov.uk
```

Comment:
Does this need link markdown/markup? Or is that added automatically?

Reply:
It's added automatically I believe